launcher/agent/agent.go

```diff
@@ -134,13 +163,20 @@ func (a *agent) Close() error {
 
 // MeasureEvent takes in a cel.Content and appends it to the CEL eventlog
 // under the attestation agent.
+// MeasureEvent measures to all Attest Roots.
 func (a *agent) MeasureEvent(event cel.Content) error {
-	return a.ar.Extend(event, &a.cosCel)
+	for _, attestRoot := range a.measuredRots {
+		if err := attestRoot.Extend(event); err != nil {
+			return err
+		}
+	}
+	return nil
 }
 
 // Attest fetches the nonce and connection ID from the Attestation Service,
 // creates an attestation message, and returns the resultant
 // principalIDTokens and Metadata Server-generated ID tokens for the instance.
+// Attest only attests to the "best" Attest Root.
```

Comment:
nit: we should define how we select the attest root.
// When possible, Attest uses the technology-specific attestation root-of-trust and otherwise falls back to the vTPM.

Reply:
done